demo/flake.nix

```
@@ -3,12 +3,15 @@
 
   inputs = {
     # Use the local flake for testing or remote flake for production.
+    # TODO: because this is a relative path input you will need to update the flake.lock file locally with `nix flake lock --update-input nixos-rk3588`
```

`nix flake lock --update-input` is deprecated.
 ```suggestion
    # TODO: because this is a relative path input you will need to update the flake.lock file locally with `nix flake update nixos-rk3588`
```